sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    triton_image_uri = "195202947636.dkr.ecr.us-west-2.amazonaws.com/tritonserver:21.08-py3"
</pre><p>use <code>sagemaker.image_uris.retrieve()</code>. Account ID, region, and version tag cannot be hard-coded. (In both notebooks)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<ul><li>Removed hard-coding</li><li>Adding logic to generate the uri based on region and <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/neo-deployment-hosting-services-container-images.html" target="_blank">account ids</a></li><li>Added error handling for unsupported regions</li></ul>